terminololgy and phrasing in Introduction (#39)
lamps-wg/csr-attestation: draft-ietf-lamps-csr-attestation.md

# Introduction

> When requesting a certificate from a Certification Authority (CA), a PKI end entity may wish to include Evidence of the security properties of its environments in which the private keys are stored in that request.

Author: ```suggestion
When requesting a certificate from a Certification Authority (CA), a PKI end entity may wish to include Evidence of the security properties of its environments in which private keys are stored.
```

> This Evidence can be appraised by authoritative entities, such as a Registration Authority (RA) or a CA as part of validating an incoming certificate request against given certificate policies.

Author: ```suggestion
This Evidence can be appraised by authoritative entities, such as a Registration Authority (RA) or a CA as part of validating an incoming certificate request against given certificate policy.
```

> This specification provides two newly defined attributes that allow for conveyance of remote attestation payload in Certificate Requests (CSR) in either PKCS#10 {{RFC2986}} or Certificate Request Message Format (CRMF) {{RFC4211}}.

Reviewer: ```suggestion
This specification defines an attribute and an extension that allow for conveyance of Evidence in Certificate Signing Requests (CSRs) in either PKCS#10 {{RFC2986}} or Certificate Request Message Format (CRMF) {{RFC4211}} payloads.
```

As outlined in the RATS Architecture {{RFC9334}}, an Attester (typically
a device) produces a signed collection of Claims that constitutes Evidence about its running environment.

> The term "attestation" is intentionally not defined in RFC 9334, but it is often taken to mean the activity of producing and appraising remote attestation Evidence.

Author: While RFC9334 does not define `attestation`, https://www.ietf.org/archive/id/draft-ietf-rats-tpm-based-network-device-attest-14.html#name-terminology-2 does. Should we take that into account?

> The term "attestation" is intentionally not defined in RFC 9334, but it is often taken to mean the activity of producing and appraising remote attestation Evidence.

Reviewer: ```suggestion
While the term "attestation" is not defined in RFC 9334, it was later defined in {{?I-D.[ietf-rats-tpm-based-network-device-attest}}, it refers to the activity of producing and appraising remote attestation Evidence.
```

> A Relying Party may consult that Evidence -- or an Attestation Result produced by a Verifier that has appraised the Evidence -- in making policy decisions about the trustworthiness of the

Reviewer: ```suggestion
A Relying Party may consult an Attestation Result produced by a Verifier that has appraised the Evidence in making policy decisions about the trustworthiness of the
```

The certificates typically contain one or more certification paths
rooted in a device manufacture trust anchor and the leaf certificate being
on the device in question; the latter is the Attestation Key that signs the evidence statement.
on the device in question; the latter is the Attestation Key that signs the Evidence statement.

This document creates two ATTRIBUTE/Attribute definitions. The first
This document specifies two ATTRIBUTE/Attribute definitions. The first
Attribute may be used to carry a set of certificates or public keys that
may be necessary to validate evidence. The second Attribute carries a
structure that may be used to carry evidence.
may be required to validate signed Evidence. The second Attribute carries a
structure that may be used to convey Evidence.

A CSR may contain one or more evidence, for example evidence
asserting the storage properties of the private key as well evidence
asserting the firmware version and other general properties
of the device, or evidence signed by certificate chains

> on different cryptographic algorithms.

Author: As I assumed that `different cryptographic algorithms` does not really add anything meaningful here, I removed it. Did I unintentionally remove meaning here?

This specification provides two newly defined attributes that allow for conveyance of remote attestation payload in Certificate Requests (CSR) in either PKCS#10 {{RFC2986}} or Certificate Request Message Format (CRMF) {{RFC4211}}.

As outlined in the RATS Architecture {{RFC9334}}, an Attester (typically
a device) produces a signed collection of Claims that constitutes Evidence about its running environment.
The term "attestation" is intentionally not defined in RFC 9334, but it is often taken to mean the activity of producing and appraising remote attestation Evidence.
A Relying Party may consult that Evidence -- or an Attestation Result produced by a Verifier that has appraised the Evidence -- in making policy decisions about the trustworthiness of the
target environment being assessed via appraisal of Evidence. {{architecture}} provides the basis to illustrate in this document how the various roles
in the RATS architecture map to a certificate requester and a CA/RA.

At the time of writing, several standard and several proprietary attestation technologies
are in use.

> This specification thereby is intended to as technology-agnostic as it is feasible with respect to implemented remote attestation technologies. This specification focuses on the transport of Evidence via CSR while making minimal assumptions about content or format of the transported Evidence.

Author: ```suggestion
This specification thereby is intended to be as technology-agnostic as it is feasible with respect to implemented remote attestation technologies. This specification focuses on the transport of Evidence via CSR while making minimal assumptions about content or format of the transported Evidence.
```
